pages/en/zkapps/how-to-write-a-zkapp.mdx

```diff
@@ -391,14 +391,22 @@ Off-chain state is not yet available.
 
 ### Testing your smart contract
 
-Testing is critical when writing smart contracts.
+Writing automated tests for your smart contract is of crucial importance. The <a href="https://jestjs.io/">Jest testing framework</a> is included in all projects created by the Mina zkApp CLI via `zk project <name>`. We recommend Jest as your project's test runner, but any testing framework can be used.
```

Comment:
Maybe worth mentioning example projects also-  
all projects created by the Mina zkApp CLI via `zk project <name>` and `zk example <name>` .

Reply:
+1  Yoni
> `zk project <name>` or `zk example <name>`

Would remove `as your project's test runner` to tighten it up while saying same thing